src/structures/Player.ts

```diff
@@ -162,20 +162,8 @@ export class Player {
      */
     public async play() {
         if (!this.queue.current) throw new ChiroError("Queue is empty to play!");
-        if (this.state == "disconnected") await this.connect();
-        
-        return await new Promise((resolve, reject) => {
-            if (this.state == "connected") this.sendPlayPost(this.queue.current).then(() => resolve(null), reject);
-            else {
-                const connectInterval = setInterval(() => {
-                    if (this.state == "connected") this.sendPlayPost(this.queue.current).then(() => resolve(null), reject);
-                    else if (this.state == "disconnected") {
-                        clearInterval(connectInterval);
-                        return reject(new ChiroError(`Player has been disconnected but has been assigned to play!`));
-                    }
-                }, 10000);
-            }
-        });
+        if (!this.connected) throw new ChiroError("The player is still not yet connected.");
+        return this.sendPlayPost(this.queue.current);
```

Comment:
the voice event sometimes late thats why i added an interval. It's better to keep it like that. It makes it easier for user. Not to mention usually user make play command with direct search.

Reply:
I have changed the code so hereafter you should await `manager.create({ ... })` where it would complete all the necessary stuff  and then i play it. I will change the README example soon.